tensorflow/c/eager/c_api_internal.h

```diff
@@ -96,13 +100,24 @@ struct TFE_TensorDebugInfo {
   std::vector<tensorflow::int64> dev_dims;
 };
 
+struct TFE_OpInferenceContext {
+  explicit TFE_OpInferenceContext(const tensorflow::OpDef* op_def)
+      : op_def(op_def) {}
+
+  const tensorflow::OpDef* op_def;  // op definition from protobuf
+  int input_idx = 0;                // index of the next input to be added
```

Comment:
Clarify the comment to say that this is distinct from EagerOperation.MutableInputs()->size() because list inputs increment this by 1 instead of their size. This is the index into the input ArgDef in `op_def`.